src/InputWithLabel.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useRef } from 'react';
+
+function InputWithLabel (props){
+
+    const inputRef = useRef();
+    useEffect(() => {
+        inputRef.current.focus();
+    },[])
```

Comment:
When you place an empty array in the dependency list, it means that the useEffect() is going to run only once. You may have noticed that when you enter a new item, you have to click back into the input field to type another todo. Remove the array and it will run every time the component renders. When you hit enter, you will be able to enter a new todo immediately.